src/utilities/utilities.hpp

```diff
@@ -206,8 +227,16 @@ struct Arguments {
   bool print_help = false;
   bool silent = false;
   bool no_abbrv = false;
+
+  int tStrategy = DEFAULT_STRATEGY;
+  size_t psoSwarmSize = DEFAULT_PSO_SWARM;
+  double psoInfG = DEFAULT_PSO_G;
+  double psoInfL = DEFAULT_PSO_L;
+  double psoInfR = DEFAULT_PSO_R;
 };
 
+
+
```

Comment:
Try to keep the changes only to the minimum required. So no 2 extra empty lines if they are not really needed!